src/components/home/elements/TxInfoModal.vue

```diff
@@ -6,6 +6,7 @@
            centered
            hide-header
            hide-footer>
+    <router-view v-if="isRouterAlive"></router-view>
```

Comment:
What's the router use for?

Reply:
It can automatically refresh the data in 'Exists in watching list' when click the button 'add to list'. Without it, you have to reopen TxInfoModal to refresh data. I think this is user-friendly.

Reply:
OK. I see. This is user-friendly.